scenario.py

```diff
@@ -292,7 +294,7 @@ def split_data_advanced(self, is_logging_enabled=True):
             shared_clusters_count = max([p.cluster_count for p in partners_with_shared_clusters])
         else:
             shared_clusters_count = 0
-        assert specific_clusters_count + shared_clusters_count <= nb_diff_labels
+        assert specific_clusters_count + shared_clusters_count <= nb_diff_labels, "Error: data samples from the initial dataset are split in clusters per data labels - ['advanced', [[7, 'shared'], [6, 'shared'], [2, 'specific'], [1, 'specific']]] means 7 shared clusters and 2 + 1 = 3 specific clusters ==> The dataset need to have >= 10 labels"
```

Comment:
You can't write "10 labels" as it depends upon the dataset configured, try to word it in a more generic way.
Also, good thing to give an example, just indicate explicitely that you are providing an example :smiley: